required/latex-lab/latex-lab-math.dtx

```diff
@@ -778,39 +916,62 @@
         \iow_newline:
         <p>\l_@@_content_hash_tl </p>
         \iow_newline:
-      }   
+      }
   }
 %    \end{macrocode}
 % \end{plugdecl}
 %
 % With luatex we automatically generate mathml with luamml if the package
-% can be loaded and unicode-math is detected. 
+% can be loaded and unicode-math is detected.
 % We start the process in the begindocument/end hook
 % so that the reading from a previous compilation can happen before!
+%
+% For other engines, for future name changes
+% and in case luamml is not loaded we provide
+% some commands
 %    \begin{macrocode}
-\sys_if_engine_luatex:T
- {
-   \file_if_exist:nT { luamml.sty }
+\cs_new_protected:Npn\@@_provide_luamml_commands:
+  {
+    \providecommand\luamml_flag_structelem:{}
+    \cs_if_free:NT \luamml_structelem:
+     {
+       \cs_set_eq:NN\luamml_structelem:\luamml_flag_structelem:
+     }
+    \providecommand\luamml_flag_process:{}
+    \cs_if_free:NT \luamml_process:
+     {
+       \cs_set_eq:NN\luamml_process:\luamml_flag_process:
+     }
+    \providecommand\luamml_flag_ignore:{}
+    \cs_if_free:NT \luamml_ignore:
      {
-       \RequirePackage  { luamml }
+       \cs_set_eq:NN\luamml_ignore:\luamml_flag_ignore:
+     }
+  }
 %    \end{macrocode}
-% Temporary (!) fixes for endarray 
 %    \begin{macrocode}
+\sys_if_engine_luatex:TF
+ {
+%    \end{macrocode}
+% Temporary (!) fixes for endarray
```

Comment:
There should be somewhere an issue that ensure that array gets an appropriate interace for this at some point.

Reply:
is there, or why "resolved"?